soc/silabs/Kconfig.defconfig

```diff
@@ -19,4 +19,7 @@ config CORTEX_M_SYSTICK
 config IDLE_STACK_SIZE
 	default 512 if SOC_GECKO_PM_BACKEND_PMGR
 
+configdefault NUM_METAIRQ_PRIORITIES
```

Comment:
This is the only way I found to enable meta-IRQ priorities. The default doesn't really belong to the SoC, but the driver doesn't have a `.defconfig` files to affect the default value.

Reply:
Yes, that didn't work. The correct definition in `kernel/Kconfig` gets precedence as it's loaded before `drivers/Kconfig`. Besides, I understood the overrides like this should always be defined in `.defconfig` files with `configdefault` keyword.

Reply:
Weird. It seems to work with `drivers/wifi/nrf_wifi/Kconfig.nrfwifi:430`.

Reply:
I also found that while trying to figure out how meta-IRQ priorities are supposed to be enabled. However, that doesn't really work as intended:
```
$ west build -p -b nrf7002dk/nrf5340/cpuapp zephyr/tests/drivers/wifi/nrf_wifi
$ grep -e CONFIG_WIFI_NRF70= -e CONFIG_NUM_METAIRQ_PRIORITIES= build/zephyr/.config
CONFIG_NUM_METAIRQ_PRIORITIES=0
CONFIG_WIFI_NRF70=y
```

Reply:
And you can't relocate `configdefault NUM_METAIRQ_PRIORITIES` in `drivers/bluetooth/hci/Kconfig.silabs`?

Reply:
My understanding is that it's discouraged, if not outright forbidden to do overlays in anything else than defconfig files, just to be able to more easily track where the redefinitions of a symbol might happen. That said, I didn't find this anywhere explicitly stated in the documentation, rather this is just the impression I've gotten in past code reviews and discussions.

Reply:
Maybe the right thing to do is to add `default 1 if BT_SILABS_EFR32` to the initial definition in `kernel/Kconfig`.

Reply:
> And you can't relocate `configdefault NUM_METAIRQ_PRIORITIES` in `drivers/bluetooth/hci/Kconfig.silabs`?

Same thing. The `configdefault` keyword needs to be before the actual definition. All `.defconfig` files are sourced in the beginning of `Kconfig.zephyr` for this reason.